lib/Twig/Test/IntegrationTestCase.php

```diff
@@ -17,27 +17,52 @@
  */
 abstract class Twig_Test_IntegrationTestCase extends PHPUnit_Framework_TestCase
 {
-    abstract protected function getExtensions();
+    /**
+     * @return string
+     */
     abstract protected function getFixturesDir();
 
     /**
-     * @dataProvider getTests
+     * @return Twig_ExtensionInterface[]
      */
-    public function testIntegration($file, $message, $condition, $templates, $exception, $outputs)
+    protected function getExtensions()
     {
-        $this->doIntegrationTest($file, $message, $condition, $templates, $exception, $outputs);
+        return array();
+    }
+
+    /**
+     * @return Twig_SimpleFilter[]
+     */
+    protected function getFilters()
+    {
+        return array();
     }
 
     /**
-     * @dataProvider getLegacyTests
-     * @group legacy
+     * @return Twig_SimpleFunction[]
      */
-    public function testLegacyIntegration($file, $message, $condition, $templates, $exception, $outputs)
```

Comment:
this should be kept IMO. We don't have legacy tests in 2.0.0 because we are cleaning all legacy layers, but we will need it again in 2.x when deprecating stuff again